doc/source/installation.rst

```diff
@@ -69,6 +69,25 @@ some benefits:
 * allows molecule installation at user level without even needing to have
   the script in ``PATH``.
 
+.. note::
+
+    We also have a continuous pre-release process which enables early feedback.
+    This pre-release is provided for testing purposes only. It is available
```

Comment:
"testing" is a loaded term when we are talking about Molecule which is a testing tool, though my brain is failing to come up with something better.

Reply:
Fair enough! Pushed a rewording which uses 'early adoption and feedback purposes only'.